protocols/knxnetip/src/main/resources/protocols/knxnetip/knxnetip.mspec

```diff
@@ -882,9 +882,9 @@
             [reserved uint 4 '0x00']
             [simple uint 4 month]
             [reserved uint 3 '0x00']
-            [simple uint 5 dayofmonth]
+            [simple uint 5 dayOfMonth]
```

Comment:
MIght as well camel case the datofweek as well.

Reply:
Ooooh ... jesus ... I think in the type PDT_DATE_TIME something really creapy is going on ... I see day of week sort of spanning across two bytes.

Reply:
Aaaah no ... it's actually not that scary ... just noticed the 4th bit is actually just a flag if the first part is present ;-)